### PR TITLE
Optimize time run

### LINKED ISSUE #136: optimize test time run

### PR DESCRIPTION
**Pull request by anhductn2001**
Close #136

### CODE REVIEW

File: tests/eibc_pfm_test.go [hunk 1]
```diff
@@ -573,23 +526,13 @@ func TestEIBCPFM_Wasm(t *testing.T) {
 
 	triggerHubGenesisEvent(t, dymension, rollapp1Param)
```
Comment (trinitys7) [inline tests/eibc_pfm_test.go:527]: Should combine 2 function into 1

File: tests/ibc_pfm_with_grace_period_test.go [hunk 2]
```diff
@@ -288,7 +281,7 @@ func TestIBCPFMWithGracePeriod_EVM(t *testing.T) {
 		err = transferTx.Validate()
 		require.NoError(t, err)
 
-		err = testutil.WaitForBlocks(ctx, 20, rollapp1)
+		err = testutil.WaitForBlocks(ctx, 10, dymension)
```
Comment (trinitys7) [inline tests/ibc_pfm_with_grace_period_test.go:284]: Should be wait for finalize

File: tests/ibc_timeout_test.go [hunk 3]
```diff
@@ -39,14 +39,15 @@ func TestIBCTransferTimeout_EVM(t *testing.T) {
...
 
 	configFileOverrides["config/dymint.toml"] = dymintTomlOverrides
-	const BLOCK_FINALITY_PERIOD = 10
+
 	modifyGenesisKV := append(
 		dymensionGenesisKV,
 		cosmos.GenesisKV{
 			Key:   "app_state.rollapp.params.dispute_period_in_blocks",
-			Value: fmt.Sprint(BLOCK_FINALITY_PERIOD),
+			Value: fmt.Sprint(10),
```
Comment (trinitys7) [inline tests/ibc_timeout_test.go:47]: Why don't we use BLOCK_FINALITY_PERIOD here?

File: tests/ibc_timeout_test.go [hunk 4]
```diff
@@ -274,14 +272,15 @@ func TestIBCTransferTimeout_Wasm(t *testing.T) {
...
```
Comment (trinitys7) [inline tests/ibc_timeout_test.go:280]: This one either, why not BLOCK_FINALITY_PERIOD?

**Review by trinitys7 — CHANGES_REQUESTED** [review]

File: tests/ibc_pfm_with_grace_period_test.go [hunk 5]
```diff
@@ -37,10 +36,13 @@ func TestIBCPFMWithGracePeriod_EVM(t *testing.T) {
...
 
-	modifyGenesisKV := append(dymensionGenesisKV, cosmos.GenesisKV{
-		Key:   "app_state.rollapp.params.dispute_period_in_blocks",
-		Value: "60",
-	})
+	modifyGenesisKV := append(
+		dymensionGenesisKV,
+		cosmos.GenesisKV{
+			Key:   "app_state.rollapp.params.dispute_period_in_blocks",
+			Value: fmt.Sprint(BLOCK_FINALITY_PERIOD),
```
Comment (ThanhNhann) [inline tests/ibc_pfm_with_grace_period_test.go:43]: the old value is 60 but `BLOCK_FINALITY_PERIOD` is 30, does this effect with our test @anhductn2001

File: tests/ibc_pfm_with_grace_period_test.go [hunk 6]
```diff
@@ -337,10 +327,12 @@ func TestIBCPFMWithGracePeriod_Wasm(t *testing.T) {
...
 	dymintTomlOverrides["empty_blocks_max_time"] = "3s"
 
-	modifyGenesisKV := append(dymensionGenesisKV, cosmos.GenesisKV{
-		Key:   "app_state.rollapp.params.dispute_period_in_blocks",
-		Value: "60",
-	})
+	modifyGenesisKV := append(dymensionGenesisKV,
+		cosmos.GenesisKV{
+			Key:   "app_state.rollapp.params.dispute_period_in_blocks",
+			Value: fmt.Sprint(BLOCK_FINALITY_PERIOD),
```
Comment (ThanhNhann) [inline tests/ibc_pfm_with_grace_period_test.go:333]: Same question here

File: tests/ibc_pfm_with_grace_period_test.go (hunk 5 above)
Comment (anhductn2001) [reply tests/ibc_pfm_with_grace_period_test.go:43]: I think it's not @ThanhNhann

File: tests/eibc_fulfillment_test.go [hunk 7]
```diff
@@ -603,7 +588,7 @@ func TestEIBCFulfillment_Wasm(t *testing.T) {
 	}
 
 	// wait a few blocks and verify sender received funds on the hub
-	err = testutil.WaitForBlocks(ctx, 5, dymension)
+	err = testutil.WaitForBlocks(ctx, 10, dymension)
```
[the remark this answers is not in the document]
Comment (anhductn2001) [reply tests/eibc_fulfillment_test.go:591]: I know waiting for 5 blocks is fine, but I'm setting all waits to a minimum of 10 blocks to ensure everything is stable.

**Review by ThanhNhann — APPROVED** [review]

**Review by trinitys7 — APPROVED** [review]

**Pull request merged**